.eslintrc-ts-frontend.json

```diff
@@ -0,0 +1,12 @@
+{
+  "extends": ["./.eslintrc-ts.json"],
+  "env": {
+    "browser": true
+  },
+  "rules": {
+    "jsx-a11y/href-no-hash": "off",
+    "react/destructuring-assignment": "off",
+    "react/no-unescaped-entities": "off",
+    "import/extensions": "off"
+  }
+}
```

Comment:
I like this reusable setting. Just want to check, are we going to miss out on some useful things by not using the `react-app` lint settings?